### ISSUE TITLE
convert to a pandas df

### ISSUE THREAD

**Issue by ratnarajsingh** [wwwiiilll/pyxlsb#12]
I'm trying to write a function to convert the sheet into a dataframe. A way exists?

**Comment by wwwiiilll** [comment]
There's no direct way available.

It's been a while since I worked with Pandas, but I'm fairly certain you can get a representation that works by collecting only the `v` key of each cells.

**Comment by chfw** [comment]
Indirectly with pyexcel-xlsbr, you can get dict out. But no type information, such as date and time.

**Comment by adamzev** [comment]
Finrod Felagund details a method in a [Stack Overflow post](https://stackoverflow.com/a/49400870/1830793).

```
import pandas as pd
from pyxlsb import open_workbook as open_xlsb

df = []

with open_xlsb('some.xlsb') as wb:
    with wb.get_sheet(1) as sheet:
        for row in sheet.rows():
            df.append([item.v for item in row])

df = pd.DataFrame(df[1:], columns=df[0])
```

**Comment by wwwiiilll** [comment]
Will be resolved with pandas-dev/pandas#29836
